skull-king-app/package-lock.json

```diff
@@ -0,0 +1,4195 @@
+{
```

Comment:
Run yarn commands not npm commands.
Please remove package-lock.json and revert changes to yarn.lock.